openverse_catalog/dags/providers/provider_api_scripts/wordpress.py

```diff
@@ -264,32 +210,34 @@ def _get_file_info(image_details):
     return image_url, height, width, filetype
 
 
-def _get_thumbnail_url(image_details):
-    return (
-        image_details.get("media_details", {})
-        .get("sizes", {})
-        .get("thumbnail", {})
-        .get("source_url")
-    )
+def _get_thumbnail_url(media_details):
+    return media_details.get("sizes", {}).get("thumbnail", {}).get("source_url")
 
 
-def _get_creator_data(image):
-    creator, creator_url = None, None
-    if author_id := image.get("author"):
-        creator = IMAGE_RELATED_RESOURCES.get("users").get(author_id, {}).get("name")
-        creator_url = IMAGE_RELATED_RESOURCES.get("users").get(author_id, {}).get("url")
-    return creator, creator_url
+def _get_author_data(image):
+    try:
+        author = image.get("_embedded", {}).get("author", [])[0]
+    except IndexError:
+        return None, None
+    author_url = author.get("url")
+    if author_url == "":
+        author_url = author.get("link")
+    return author.get("name"), author_url
 
 
 def _get_title(image):
     if title := image.get("content", {}).get("rendered"):
-        title = html.fromstring(title).text_content()
+        try:
+            title = html.fromstring(title).text_content()
+        except UnicodeDecodeError as e:
+            logger.warning(f"Can't save the image's title ('{title}') due to {e}")
+            return None
```

Comment:
There were some images including emojis in this field that we treat as the title crashing the script. I'm not sure how to handle emojis yet.